code/mob/living/critter/ai/flock/flocktasks.dm

```diff
@@ -905,3 +905,75 @@ butcher
 
 /datum/aiTask/succeedable/butcher/on_reset()
 	has_started = FALSE
+
+///////////////////////////////////////////////////////////////////////////////////////////////////////////
+// STARE AT BIRDS
+// targets: any bird critters
+// precondition: cooldown
+/datum/aiTask/sequence/goalbased/stare
+	name = "observing"
+	weight = 1 //same as wander
+
+/datum/aiTask/sequence/goalbased/stare/New(parentHolder, transTask)
+	..(parentHolder, transTask)
+	add_task(holder.get_instance(/datum/aiTask/succeedable/stare_at_bird, list(holder)))
+
+/datum/aiTask/sequence/goalbased/stare/precondition()
+	. = FALSE
+	if(!GET_COOLDOWN(holder.owner, "bird_staring"))
+		. = TRUE
+
+/datum/aiTask/sequence/goalbased/stare/get_targets()
+	. = list()
+	for(var/mob/living/critter/C in view(max_dist, holder.owner)) //I'm not handling obj/critter birds. i'm just not. deal with it.
+		if(istype(C,/mob/living/critter/small_animal/bird) || istype(C,/mob/living/critter/small_animal/ranch_base/chicken))
+			. += C
+
+	. = get_path_to(holder.owner, ., max_dist*2, 1)
+
+////////
+
+/datum/aiTask/succeedable/stare_at_bird
+	name = "stur at burd"
+	var/has_started = FALSE
+
+/datum/aiTask/succeedable/stare_at_bird/failed()
+	return !IN_RANGE(holder.owner, holder.target, 1)
+
+/datum/aiTask/succeedable/stare_at_bird/succeeded()
+	return has_started
+
+/datum/aiTask/succeedable/stare_at_bird/on_tick()
+	if(!has_started)
+		var/mob/living/critter/target = holder.target
+		var/mob/living/critter/flock/drone/F = holder.owner
+		if(target && IN_RANGE(holder.owner, target, 1))
```

Comment:
should probably make these ifs early returns

Reply:
yeah, probably. will fix